.travis.yml

```diff
@@ -1,16 +0,0 @@
-language: node_js
-sudo: required
-before_script:
-  - npm install -g polymer-cli
-  - polymer install --variants
-node_js: stable
-addons:
-  firefox: latest
-  apt:
-    sources:
-      - google-chrome
-    packages:
-      - google-chrome-stable
-script:
-  - xvfb-run polymer test
-dist: trusty
```

Comment:
Is `.travis.yml` no longer needed?

Reply:
I couldn't see it being used by anything, so removed it. Do you want it back in?

Reply:
Ah okay, will add it back 👍